libs/application/template-api-modules/src/lib/modules/templates/children-residence-change/children-residence-change.service.ts

```diff
@@ -104,13 +104,14 @@ export class ChildrenResidenceChangeService {
 
     participants.push(parentA, parentB)
 
+    const durationType = answers.durationType as string
     const extraData = {
       interviewRequested: answers.interview,
       reasonForChildrenResidenceChange: answers.residenceChangeReason ?? '',
       transferExpirationDate:
-        answers.selectDuration[0] === 'permanent'
-          ? answers.selectDuration[0]
-          : formatDate(answers.selectDuration[1]),
+        durationType === 'temporary' && answers.durationDate
```

Comment:
Since we are doing this check in a couple of places currently then we could add a helper function for it. Something like `IsDurationTemporary' which would return a boolean. Just a thought not a must.

Reply:
Keeping as is for now.